README.md

```diff
@@ -10,14 +18,14 @@
 [![brew Downloads](https://img.shields.io/homebrew/installs/dy/openvino)](https://formulae.brew.sh/formula/openvino)
  </div>
 
-Welcome to OpenVINO™, an open-source software toolkit for optimizing and deploying deep learning models.
 
 - **Inference Optimization**: Boost deep learning performance in computer vision, automatic speech recognition, generative AI, natural language processing with large and small language models, and many other common tasks.
 - **Flexible Model Support**: Use models trained with popular frameworks such as TensorFlow, PyTorch, ONNX, Keras, and PaddlePaddle. Convert and deploy models without original frameworks.
```

Comment:
Please explicitly mention capability to take models built with `transformers` and `diffusers` from HuggingFace Hub with the help of optimum-intel. IMO, it is a more eye catching feature than just mentioning DL frameworks.

Reply:
thanks, added!